src/udev/udevadm-hwdb.c

```diff
@@ -461,6 +461,7 @@ static int import_file(struct udev *udev, struct trie *trie, const char *filenam
         FILE *f;
         char line[LINE_MAX];
         struct udev_list match_list;
+        int ret = 0;
```

Comment:
`r` would be better than `ret`, because we use `r` for this role elsewhere.

Reply:
Done.